src/control/SimBarrettFingerPositionCommandExecutor.cpp

```diff
@@ -153,7 +156,7 @@ void BarrettFingerPositionCommandExecutor::step(double _timeSincePreviousCall)
   if (mDistalOnly)
     return;
 
-  // Check proximal collision 
+  // Check proximal collision
   bool proximalCollision = mCollisionDetector->collide(
     mProximalCollisionGroup.get(), mCollideWith.get(),
     mCollisionOptions, &collisionResult);
```

Comment:
This is not added by this PR, but we don't need to pass the collision result parameter since we only check whether there is a collision or not. By passing `nullptr` (or not passing), we could save time to compute the exact contact information such as point, normal, and penetration depth.